src/scss/components/_picture.scss

```diff
@@ -39,6 +39,12 @@
     }
   }
 
+  &--media {
+    max-width: 30px;
+    max-height: 30px;
```

Comment:
@iondrimba we had a bug using max-width on our new page of public request. Flexbox calculates this using some strange logic and the image inside stay with a wrong width, like the screenshot bellow:

![screen shot 2017-04-27 at 11 52 17 pm](https://cloud.githubusercontent.com/assets/1279783/25512495/91cfad38-2ba4-11e7-8520-1e7b4568699c.png)

To resolve this, we can apply a `min-width: 30px` instead `max-width`. Or we can just add `width: 100%` with this `max-width`.